_components/header/header.html

```diff
@@ -30,11 +30,6 @@
 
 <h2>When to use a header component</h2>
 <ul class="usa-content-list">
-  <li>
-    <strong>Use the gov banner on all federal websites.</strong> We recommend
-    that all federal government websites include the “official government website”
-    banner and a logo or site name.
-  </li>
```

Comment:
> **Note**
> Removed notes about the banner component on the header page because the content seemed outdated and I worried it might lead to confusion.